cmd/influx/organization.go

```diff
@@ -356,6 +334,7 @@ type OrganizationMembersAddFlags struct {
 var organizationMembersAddFlags OrganizationMembersAddFlags
 
 func organizationMembersAddF(cmd *cobra.Command, args []string) error {
+	ctx := context.Background()
```

Comment:
nit, can we move this closer to where its being used, say like 364? keep it close, line of sight 🤷‍♂